awx/network_ui/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+
+class Device(models.Model):
+
+    device_id = models.AutoField(primary_key=True,)
+    topology = models.ForeignKey('Topology',)
+    name = models.CharField(max_length=200, blank=True)
+    x = models.IntegerField()
+    y = models.IntegerField()
+    id = models.IntegerField()
+    device_type = models.CharField(max_length=200, blank=True)
+    interface_id_seq = models.IntegerField(default=0,)
+    host_id = models.IntegerField(default=0,)
```

Comment:
fk reference to a host?

Reply:
I've changed host_id and inventory_id to a ForeignKey field.